ansible/roles/mn-evo-services/templates/docker-compose.yml.j2

```diff
@@ -79,10 +79,28 @@ services:
       - DASHCORE_P2P_HOST={{ private_ip }}
       - DASHCORE_P2P_PORT={{ dashd_port }}
       - DASHCORE_P2P_NETWORK={{ dash_network }}
-      - DASHDRIVE_RPC_HOST=drive_api
-      - DASHDRIVE_RPC_PORT=6000
+      - DRIVE_RPC_HOST=drive_api
+      - DRIVE_RPC_PORT=6000
       - NETWORK={{ dash_network }}
 
+  insight_api:
+    image: {{ insight_image }}
+    container_name: insight_api
+    restart: always
+    ports:
+      - 3002:3001
+    volumes:
+      - {{ mn_evo_services_path }}/dashcore-node.json:/root/.dashcore/dashcore-node.json
+
+  # in order to support obsolete URL path (/insight-api-dash)
```

Comment:
Do we know what is still using this old path, and can we change those instead?

Reply:
Yes, we could but how we will force our users to update this software?

Reply:
Easy -- by not serving this pass-thru route and forcing them to change their paths. This is a band-aid fix, and they should be updating, instead of us providing this band-aid, which:

1) is extra complexity for us, and
2) provides no incentive for anyone to ever change the path to the new Insight API path (which has been around for a while now)

Anyway, this isn't supposed to serve the entire internet via Insight, right? This is just for internal use for DAPI? Or under which scenarios is Insight being used in this repo?

Reply:
Honestly, I do not like this strategy and "break compatibility" without any announcements. 

In this setup, We are trying to reproduce the mainnet, so Insight API is not just for DAPI, but for other 3rd party needs.

Reply:
It will never be fixed if you don't fix it now. Honestly I don't think there's anyone using this but us, so I don't understand your argument for not fixing this now before the "band-aid" gets widespread. Then people really will start using the wrong path.

I think we have an opportunity to get this right early on, and we're doing something the wrong way by adding extra complexity and adding extra paths which really shouldn't exist anyway. I would be _very_ surprised if anyone is using this path at all right now. And this has been gone in Insight-API for months.

Can you tell me, what users do you think are using this to deploy Insight, anywhere? And what consumers (apps, etc.) are using this `insight-api-dash` path? I don't think there are any, but maybe you know of some that I am not aware of.

Reply:
@nmarley i've tried to remove this endpoint, and there were problems in mobile clients.
I think that we should develop strategy to deprecate endpoint and this is out of scope of this pull request. Current setup is like this and we should maintain it. If we need to remove this endpoint (which i agree we need), so we should launch some deprecation process to get rid of this url, which will involve official announcement in the company, outside of the company, measuring number of current requests to this endpoint and comparing number of requests over time. And in few month after this process started we can safely remove this endpoint, because everybody who wanted - already updated their software to use actual endpoint.

Reply:
Ok, makes sense. Let's do that then.